src/ui/popup.js

```diff
@@ -52,7 +52,10 @@ export type PopupOptions = {
  *   - an object of {@link Point}s specifing an offset for each anchor position
  *  Negative offsets indicate left and up.
  * @param {string} [options.className] Space-separated CSS class names to add to popup container
- * @param {string} [options.maxWidth] A string that sets the CSS property of the popup's maxWidth in pixels, eg "300px"
+ * @param {string} [options.maxWidth='240px'] -
+ *  A string that sets the CSS property of the popup's maximum width, eg `'300px'`.
+ *  If you want a fluid container which sizes to the inner size, you can set this property to `'none'`.
```

Comment:
How about something like "To ensure the popup resizes to fit its content, set this property to `'none'`"?